cost/google/unutilized_ip_addresses/google_unutilized_ip_addresses.pt

```diff
@@ -0,0 +1,193 @@
+  end
+  result do
+    encoding "json"
+    collect jmes_path(response,"items[*]") do
+      field "address", jmes_path(col_item,"address")
+      field "addressType", jmes_path(col_item,"addressType")
+      field "creationTimestamp", jmes_path(col_item,"creationTimestamp")
+      field "description", jmes_path(col_item,"description")
+      field "id", jmes_path(col_item,"id")
+      field "name", jmes_path(col_item,"name")
+      field "purpose", jmes_path(col_item,"purpose")
+      field "selfLink", jmes_path(col_item,"selfLink")
+      field "status", jmes_path(col_item,"status")
+      field "subnetwork", jmes_path(col_item,"subnetwork")
+    end
+  end
+end
+
+datasource "ds_sanitize_data" do
+  run_script $js_sanitize_data, $ds_addresses
+end
+
+###############################################################################
+# Scripts
+###############################################################################
+
+script "js_regions_map", type: "javascript" do
+  result "regions_map"
+  code <<-EOS
+    var regions_map=[]
+    regions_map.push(
+      {"region": "asia-east1"},
+      {"region": "asia-east2"},
+      {"region": "asia-northeast1"},
+      {"region": "asia-northeast2"},
+      {"region": "asia-south1"},
+      {"region": "asia-southeast1"},
+      {"region": "australia-southeast1"},
+      {"region": "europe-north1"},
+      {"region": "europe-west1"},
+      {"region": "europe-west2"},
+      {"region": "europe-west3"},
+      {"region": "europe-west4"},
+      {"region": "europe-west6"},
+      {"region": "northamerica-northeast1"},
+      {"region": "southamerica-east1"},
+      {"region": "us-central1"},
+      {"region": "us-east1"},
+      {"region": "us-east4"},
+      {"region": "us-west1"},
+      {"region": "us-west2"}
+    )
+  EOS
+end
+
+script "js_get_addresses", type: "javascript" do
+  result "request"
+  parameters "project","region"
+  code <<-EOS
+    request = {
+      "auth": "auth_google",
+      "host": "www.googleapis.com",
+      "verb": "GET",
+      "path": "/compute/v1/projects/"+project+"/regions/"+region+"/addresses",
+      "headers": {
+        "User-Agent": "RS Policies",
+        "Content-Type": "application/json"
+      }
+      "query_params": {
+        "filter": 'status!="IN_USE"'
+      }
+    }
+  EOS
+end
+
+script "js_sanitize_data", type: "javascript" do
+  result "data"
+  parameters "ds_addresses"
+  code <<-EOS
+    data = []
+    for (i = 0; i < ds_addresses.length; i++) {
+      if (ds_addresses[i].address != null){
+        data.push(ds_addresses[i])
+      }
+    }
+  EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_unutilized_addresses" do
+  validate_each $ds_sanitize_data do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing Google unutilized IP Addresses"
```

Comment:
Suggest this read something like this: "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Google Unutilized IP Addresses Discovered"